classes/editor_framework.php

```diff
@@ -0,0 +1,178 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * \mod_hvp\framework class
+ *
+ * @package    mod_hvp
+ * @copyright  2016 Joubel AS
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace mod_hvp;
+
+defined('MOODLE_INTERNAL') || die();
+
+require_once __DIR__ . '/../autoloader.php';
+
+/**
+ * Moodle's implementation of the H5P Editor framework interface.
+ *
+ * @package    mod_hvp
+ * @copyright  2016 Joubel AS
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class editor_framework implements \H5peditorStorage {
+
+    /**
+     * Empty contructor.
+     */
+    function __construct() { }
+
+    /**
+     * Implements getLanguage().
```

Comment:
I know the more descriptive comment belongs to the class that is being implemented, but I'd still like to have a bit more informative comments here also.

If I were a Moodle developer who knows nothing about H5P, I would e.g. in this case guess that this function gets the language code that the logged in user has set as their UI language from personal settings in Moodle.